texasholdem/game/game.py

```diff
@@ -1349,7 +1350,8 @@ def copy(self, shuffle: bool = True):
         game.start_hand()
 
         for i, cards in self.hands.items():
-            game.hands[i] = cards.copy()
+            if cards_players_to_keep and i in cards_players_to_keep:
+                game.hands[i] = cards.copy()

```

Comment:
I think I would prefer to have the default be that all the player cards are the same. Here, it's that all the cards are different.

Reply:
@LucasColas Did your latest change do anything for this?